rdbms/src/main/java/org/spine3/server/storage/jdbc/command/query/CommandStorageQueryFactory.java

```diff
@@ -91,7 +94,7 @@ public InsertCommandQuery newInsertCommandQuery(CommandId id, CommandStorageReco
      * @param id        command id
      * @param record    updated record state
      */
-    public UpdateCommandQuery newUpdateCommandQuery(CommandId id, CommandStorageRecord record){
+    public Query<Void> newUpdateCommandQuery(CommandId id, CommandStorageRecord record){
```

Comment:
Pls fix the layout when you actually make changes. This is much faster than writing comments and trying to find all such places to fix the layout.